tests/test_extensions/test_highlight.py

```diff
@@ -31,6 +31,35 @@ def test_guess(self):
         )
 
 
+class TestHighlightGuessBlock(util.MdCase):
+    """Test that highlighting works with guessing for block."""
+
+    extension = ['pymdownx.highlight', 'pymdownx.superfences']
+    extension_configs = {
+        'pymdownx.highlight': {
+            'guess_lang': "block"
```

Comment:
We are testing things like `block` or `inline`, but what we really need are tests that also test that when `block` is `True`, inline code does not use guessing and vice versa. That is the only way to ensure the feature is working as expected.